File: src/ui/asq-ui/src/app/_models/domain/notification-dto.ts
import { BaseDomainDto } from './base-domain-dto';

export class NotificationDto extends BaseDomainDto{
    public userId: number;
    public seen: boolean;
    public title: string;
    public message: string;
    public imgUrl: string;
    public videoUrl: string;
    public meetingUrl: string;
    public extMeetingUrl: string;
    public notifcationType: number;

    public NotifciationDto() { }
}

File: src/lib/models/DTO/Domain/NotificationDto.cs
namespace DTO.Domain
{
    public class NotificationDto: BaseDomainDto
    {
        public long UserId{ get; set; }
        public bool Seen{ get; set; }
        public string Title{ get; set; }
        public string Message{ get; set; }
        public string ImgUrl{ get; set; }
        public string VideoUrl{ get; set; }
        public string MeetingUrl{ get; set; }
        public string ExtMeetingUrl{ get; set; }
        public short NotificationTypeId{ get; set; }
    }
}
